### PR TITLE
Created the rails project with postgresql as database

### LINKED ISSUE #1: feature#1/create-rails-project
title: create-rails-project
description: create the project with postgresql as database

### PR DESCRIPTION
**Pull request by claunicole**
### Description
- Created the rails project
- Created the database with postgresql

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  adapter: postgresql
+  encoding: unicode
+  username: claunicole
+  # For details on connection pooling, see Rails configuration guide
+  # https://guides.rubyonrails.org/configuring.html#database-pooling
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+
+development:
+  <<: *default
+  database: reloj_checador_development
```
Comment (JuanequeX) [inline config/database.yml]: this file will not to be able on your repositorie, please add it on your `.gitignore` and add a `database.yml.example`

**Review by JuanequeX — CHANGES_REQUESTED** [review]

**Review by juperameza — CHANGES_REQUESTED** [review]
Agree with @JuanequeX

File: config/database.yml (hunk 1 above)
Comment (YilKanda) [reply config/database.yml]: I agree with @JuanequeX

**Review by YilKanda — CHANGES_REQUESTED** [review]

File: config/database.yml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+#     url: <%= ENV["MY_APP_DATABASE_URL"] %>
+#
+# Read https://guides.rubyonrails.org/configuring.html#configuring-a-database
+# for a full overview on how database connection configuration can be specified.
+#
+production:
+  <<: *default
+  database: reloj_checador_production
+  username: reloj_checador
+  password: <%= ENV["RELOJ_CHECADOR_DATABASE_PASSWORD"] %>
```
Comment (YilKanda) [inline config/database.yml]: Remove the database.yml file, clearing the git cache.
`
git rm --cached config/database.yml`

**Review by YilKanda — CHANGES_REQUESTED** [review]

**Review by juperameza — APPROVED** [review]
LGTM

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+!/tmp/storage/
+!/tmp/storage/.keep
+
+/public/assets
+
+# Ignore master key for decrypting credentials and more.
+/config/master.key
+
+# Hide database.yml for credentials
+/config/database.yml
```
Comment (JuanequeX) [inline .gitignore]: Add a blank line on the end of this file. please!

**Review by JuanequeX — CHANGES_REQUESTED** [review]

**Review by willcab — APPROVED** [review]
looks good for me

**Review by juperameza — APPROVED** [review]
LGTM

**Review by JuanequeX — APPROVED** [review]
LGTM, Good job team!

**Review by LuisFLara — APPROVED** [review]
LGTM ✅

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by LuisFLara — APPROVED** [review]
LGTM

File: .gitignore (hunk 3 above)
Comment (claunicole) [reply .gitignore]: Done

**Pull request merged**